### PR TITLE
Printing Shortest Common Supersequence in C++

### PR DESCRIPTION
**Pull request by Meghag5**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
yes

### Description
This problem will be solved using dp.
We will be given two strings and we have to find the shortest
common superstring b/w two and will have to print it's superstring.
Input -
Enter string X and string Y -
abcdaf
aebcf
Output -
Shortest Common Subsequence -
aebcdaf

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#5758

### CODE REVIEW

**Review by ankitaggarwal23 — COMMENTED** [review]
Problem Already opened by someone before you opened  #5767

**Comment by Meghag5** [comment]
hey @ankitaggarwal23 in this problem i am printing the string and in above problem the length was returned 
Pls check

**Comment by ankitaggarwal23** [comment]
That person also doing the same thing. Have to check before opening an issue

**Comment by Meghag5** [comment]
Screenshot 2021-04-14 at 3 25 50 PM
she is printing the length and I am printing string
Screenshot 2021-04-14 at 3 27 48 PM

code is different for the both

**Comment by ankitaggarwal23** [comment]
> Screenshot 2021-04-14 at 3 25 50 PM
> 
> she is printing the length and I am printing string
> Screenshot 2021-04-14 at 3 27 48 PM
> 
> code is different for the both

Oh okay Do one thing Dm me this link on discord `Ankit Aggarwal#2811` I will clarify with that participant because will let you know by evening.

**Comment by ankitaggarwal23** [comment]
> Screenshot 2021-04-14 at 3 25 50 PM
> 
> she is printing the length and I am printing string
> Screenshot 2021-04-14 at 3 27 48 PM
> 
> code is different for the both

I found that the problem she chosen is already done by you so I closed their issue. You can continue .

**Comment by Meghag5** [comment]
Thank you so much 
pls review it @ankitaggarwal23

File: C-Plus-Plus/dp/Printing_Shortest_Common_Supersequence.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+/*
+Input -
+Enter string X and string Y -
+abcdaf
+aebcf
+Output -
+Shortest Common Subsequence -
+aebcdaf
+Time Complexity - O(m*n)
```
Comment (bhav09) [inline C-Plus-Plus/dp/Printing_Shortest_Common_Supersequence.cpp]: Please write the time complexity above Sample IO

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

**Pull request merged**